visualizer/parseTree.js

```diff
@@ -662,6 +662,7 @@
   // Refresh the parse tree after attempting to parse the input.
   ohmEditor.addListener('parse:input', function(matchResult, trace) {
     $('#bottomSection .overlay').style.width = 0;  // Hide the overlay.
+    ohmEditor.parseTree.rootTrace = trace;
```

Comment:
I just pushed a change that refactors parseTree.js a bit to avoid passing around the rootTrace everywhere -- hopefully this still works you, let me know if you have any problems.